app/resources/documentation/manual/index.md

```diff
@@ -1205,10 +1205,10 @@ In this preference pane, you can change the keybaord shortcuts used in TrenchBro
 If you open the preference dialog when a map is currently opened, the list of shortcuts will contain additional entries depending on the loaded entity configuration file and the game configuration file. For each entity and special brush or face types, the following keyboard shortcuts are available.
 
 * **Entity**
-	- `View Filter > Toggle CLASSNAME visible` to toggle entities with this classname visible and invisible ([more info](#filtering_rendering_options))
```

Comment:
This is also a mistake, shouldn't delete the `#`

Reply:
Fixed.